Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp

```diff
@@ -1185,6 +1185,19 @@ int WebGLRenderingContextBase::drawingBufferHeight() const
     return m_context->getInternalFramebufferSize().height();
 }
 
```

Comment:
Now that I remember:
You could also call the `setDrawingBufferColorSpace(m_drawingBufferColorSpace)` in `initializeNewContext()`.

If you want, you can add a test where we set the colorspace to p3, lose the context, restore the context and draw and expect that the color space is the correctly retained as  p3

Reply:
> Now that I remember: You could also call the `setDrawingBufferColorSpace(m_drawingBufferColorSpace)` in `initializeNewContext()` before `reshape` call.

I'll try, thank you.

> If you want, you can add a test where we set the colorspace to p3, lose the context, restore the context and draw and expect that the color space is the correctly retained as p3

Good idea, will do.

> > Note about the "Style" failure: It's about missing spaces between braces, but that's consistent with other functions in InspectorCanvas.cpp.
> 
> I think it may be path of least resistance to format as the style checker tells you.
> 
> For things that are style checker bugs, you can add `// NOLINT`. However, this seems like the other functions are not following the standard.

To confirm/clarify: You'd prefer I satisfy the style checker in this one spot, rather than keeping the existing style from the 53 other similar lines? (Or should I fix the style in all those other lines as well?)

And I see some mac-wk2 failures, I'll need to fix those.